pkg/config/parameters.go

```diff
@@ -239,13 +221,7 @@ func (t ProtocolParameters) Validate() error {
 }
 
 // ServerParameters holds the configuration for the Contour xDS server.
-type ServerParameters struct {
-	// Defines the XDSServer to use for `contour serve`.
-	// Defaults to "envoy"
-	// Deprecated: this field will be removed in a future release when
-	// the `contour` xDS server implementation is removed.
-	XDSServerType ServerType `yaml:"xds-server-type,omitempty"`
-}
+type ServerParameters struct{}
```

Comment:
this is now empty but maybe useful in the future? for migrating to ADS?

Reply:
Yeah, I think that's why I left it in place, may be useful to hold future server-level params.